neo/SDK/RPC/Model/GetBlockHeader.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Neo.SDK.RPC.Model
+{
+    public class GetBlockHeader
+    {
+        [JsonProperty(PropertyName = "hash")]
```

Comment:
You should use the current Json implementation

Reply:
Hi Shargon,
The current Json implemention needs to implement the ToJson and FromJson method for each model class. And Newtonsoft Json has been refered by Akka 
![image](https://user-images.githubusercontent.com/4393234/59909226-c0cdab00-9441-11e9-99e1-992776cb1e34.png)

Reply:
We design the Json implementation for determinacy. So we should use it in everywhere.

Reply:
OK, got it

Reply:
Change to current Json implementation